projects/crypto-currency/destemidos/Api/routes/singin.js

```diff
@@ -1,7 +1,32 @@
 import app from "./configs/app.js"
+import dotenv from 'dotenv'
+import jwt from "jsonwebtoken"
+import bcrypt from "bcrypt"
+import { Mongoose, UserSchema } from './configs/db.js'
 
-app.get('/login', (req,res)=>{
-    res.send('Fazer o login')
+dotenv.config()
+
+app.post('/singin', async (req, res) => {
+ 
+  const{ email, password } = req.body
+  const userModel = Mongoose.model('users', UserSchema, 'users')
+
+  const Users = await userModel.findOne({ email })
```

Comment:
o findOne só vai retornar apenas 1 user, oque acha de fazer essa alteração?
```suggestion
  const user = await userModel.findOne({ email })
```

Reply:
feito a substituição